File: C++/Armstrong Numbers.cpp
class Solution {
  public:
    string armstrongNumber(int n) {
        int length= 0;
        int sum = 0;
        int m = n;
        while(m!=0){
          m =m/10 ;
          length++;
        }
        m = n;
        while(m!=0){
            int digit=m%10;
            sum = sum +int(pow(digit,length));
            m=m/10;
        }
        if(sum==n){return "true";}
        return "false";
    }
};

File: JAVA/Armstrong Numbers.java
class Solution {
    static String armstrongNumber(int n){
        int sumOfCubes = 0;
        int number = n;
        while (number > 0) {
            int modulus = number % 10;
            sumOfCubes += (modulus * modulus * modulus);
            if (sumOfCubes > n) {
                return "false";
            }
            number = number / 10;
        }
        if (n == sumOfCubes) {
            return "true";
        }
        return "false";
    }
}
